### PR TITLE
Stamp scalac jars

### PR DESCRIPTION
**Pull request by liucijus**
ScalacWorker stamps jars with target label

### CODE REVIEW

File: src/java/io/bazel/rulesscala/jar/JarCreator.java [hunk 1]
```diff
@@ -265,8 +266,18 @@ public static void buildJar(String[] args) throws IOException {
       manifestFile = args[1];
       idx = 2;
     }
+
+    String targetLabel = null;
```
Comment (simuons) [inline src/java/io/bazel/rulesscala/jar/JarCreator.java:270]: Probably usage message needs to be updated. Current impl requires flags in specific order ie `-m ... -t ...` maybe need to be more flexible here?

Comment (liucijus) [reply src/java/io/bazel/rulesscala/jar/JarCreator.java:270]: I tried to do a minimal change here - it's a copy/paste from bazel: https://github.com/bazelbuild/bazel/blob/master/src/java_tools/buildjar/java/com/google/devtools/build/buildjar/jarhelper/JarCreator.java

I think better solution would be to research and find a way to depend on the original code and extend/override its functionality. But that's probably too big for this PR.

Comment (liucijus) [reply src/java/io/bazel/rulesscala/jar/JarCreator.java:270]: Added info to usage string

**Review by ittaiz — APPROVED** [review]
one question- is the name “Target-Label” also what java uses?

**Comment by liucijus** [comment]
> one question- is the name “Target-Label” also what java uses?

https://github.com/bazelbuild/bazel/blob/master/src/java_tools/buildjar/java/com/google/devtools/build/buildjar/jarhelper/JarHelper.java#L65

**Comment by ittaiz** [comment]
👍 looks good. Resolve the conflict and merge

**Pull request merged**